views/js/controller/creator/creator.js

```diff
@@ -66,18 +68,38 @@ define([
     previewerFactory,
     loggerFactory,
     subsectionView
-){
+) {
     'use strict';
     const logger = loggerFactory('taoQtiTest/controller/creator');
 
+    const filterVisiblePresets = presetGroups => {
```

Comment:
niptick: can you please document the function

Reply:
Thanks, done